### ISSUE TITLE
json-schema-ref-parser dependency is deprecated Please switch to @apidevtools/json-schema-ref-parser

### ISSUE THREAD

**Issue by fmazeiras** [json-schema-faker/json-schema-faker#828]
json-schema-ref-parser  6.1.0 is deprecated Please switch to @apidevtools/json-schema-ref-parser

**Comment by mihaimacarie98** [comment]
It's even vulnerable: https://ossindex.sonatype.org/vulnerability/CVE-2024-29651?component-type=npm&component-name=%40apidevtools%2Fjson-schema-ref-parser&utm_source=dependency-track&utm_medium=integration&utm_content=v4.11.7

**Comment by pateketrueke** [comment]
Yeah, I know we need to change it, last time I tried but it broke a lot... probably next weeks I'll try again!
